packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -109,6 +121,15 @@ export const CommandMenu = () => {
     [toggleCommandMenu, setSearch],
   );
 
+  useScopedHotkeys(
+    'esc,Esc,Escape,escape',
+    () => {
+      closeCommandMenu();
+    },
+    AppHotkeyScope.CommandMenuOpen,
```

Comment:
Not sure you need to add another hotKeyScope for this feature. Why not using `AppHotkeyScope.CommandMenu`?

Reply:
Ok, I have my answer for this one -> https://github.com/twentyhq/twenty/issues/3150#issuecomment-1869066747
Resolving the comment